scale/docs/architecture/strike.rst

```diff
@@ -4,116 +4,193 @@
-workspace that should ingest the file. To see all of the options for a Strike process's configuration, please refer to
-the Strike Configuration Specification below.
+When a new file is copied into the monitored workspace, its file name is checked against a number rules using regular
+expressions configured for that Strike job. When the first rule that matches the new file's name is reached, that rule's
+other fields indicate how Strike should handle the file, such as tagging it with data type tags or moving the file to a
+new location in a different workspace.
 
 .. _architecture_strike_spec:
 
-Strike Configuration Specification Version 1.0
+Strike Configuration Specification Version 2.0
 ------------------------------------------------------------------------------------------------------------------------
 
 A valid Strike configuration is a JSON document with the following structure:
  
 .. code-block:: javascript
 
    {
-      "version": "1.0",
-      "mount": STRING,
-      "transfer_suffix": STRING,
-      "files_to_ingest": [
-          {
-              "filename_regex": STRING,
-              "data_types": [
-                 STRING,
-                 STRING
-              ],
-              "workspace_path": STRING,
-              "workspace_name": STRING
-          }
-      ]
+       "version": "2.0",
+       "workspace": STRING,
+       "monitor": {
+           "type": STRING
+       },
+       "files_to_ingest": [
+           {
+               "filename_regex": STRING,
+               "data_types": [
+                   STRING,
+                   STRING
+               ],
+               "new_workspace": STRING,
+               "new_file_path": STRING
+           }
+       ]
    }
 
 **version**: JSON string
 
     The *version* is an optional string value that defines the version of the configuration used. This allows updates to
     be made to the specification while maintaining backwards compatibility by allowing Scale to recognize an older
     version and convert it to the current version. The default value for *version* if it is not included is the latest
-    version, which is currently 1.0. It is recommended, though not required, that you include the *version* so that
+    version, which is currently 2.0. It is recommended, though not required, that you include the *version* so that
     future changes to the specification will still accept your Strike configuration.
 
-**mount**: JSON string
+**workspace**: JSON string
 
-    The *mount* field is a required string that specifies the NFS host and path that should be mounted in order to
-    access the monitored directory (format is *host:/file/path*).
+    The *workspace* field is a required string that specifies the name of the workspace that is being monitored. The
+    type of the workspace (its broker type) will determine how incoming files can be monitored.
```

Comment:
Is this true? Below it suggests the **monitor** field determines how things are monitored. This should be clarified.